### ISSUE TITLE
Does not work on Linux

### ISSUE THREAD

**Issue by NarryG** [ScrimCreations/VRChat-EAC-Bypass#1]
This doesn't work on the Proton build of VRChat due to dependencies on the Windows API. 
Please make this POSIX compliant.

**Comment by ScrimCreations** [comment]
I will don't worry!

**Comment by Fatalution** [comment]
Yeah, really need this POSIX compliant. Can't leave Unix users missing out on this really valuable tool

**Comment by Kamilake** [comment]
It looks like it will take some time to make this very large, great and complex open source Linux compatible version
